src/com/zetcode/Board.java

```diff
@@ -85,26 +94,20 @@ public void paintComponent(Graphics g) {
     }
 
     private void drawObjects(Graphics2D g2d) {
-
-        g2d.drawImage(ball.getImage(), ball.getX(), ball.getY(),
-                ball.getImageWidth(), ball.getImageHeight(), this);
-        g2d.drawImage(paddle.getImage(), paddle.getX(), paddle.getY(),
-                paddle.getImageWidth(), paddle.getImageHeight(), this);
+        
+        ball.draw(g2d);
+        paddle.draw(g2d);
 
         for (int i = 0; i < Commons.N_OF_BRICKS; i++) {
-
             if (!bricks[i].isDestroyed()) {
-
-                g2d.drawImage(bricks[i].getImage(), bricks[i].getX(),
-                        bricks[i].getY(), bricks[i].getImageWidth(),
-                        bricks[i].getImageHeight(), this);
+                bricks[i].draw(g2d);
             }
         }
     }
 
     private void gameFinished(Graphics2D g2d) {
 
-        var font = new Font("Verdana", Font.BOLD, 18);
```

Comment:
If my understanding is correct, var is part of a newer version of Java.  Why go backwards to support older versions?